va/validation-authority_test.go

```diff
@@ -627,7 +612,7 @@ func TestDNSValidationBadDNSSEC(t *testing.T) {
 
 func TestDNSValidationNoServer(t *testing.T) {
```

Comment:
Since there is still code that (randomly) picks a server from a list I think the no server/duplicate server tests should still be useful.